man/man1/arcstat.1

```diff
@@ -0,0 +1,482 @@
+.sp
+.ne 2
+
+.\"
+.sp
+.ne 1
+.na
+\fBc \fR
+.ad
+.RS 14n
+ARC Target Size
+.RE
+
+.sp
+.ne 2
+.na
+\fBdh% \fR
+.ad
+.RS 14n
+Demand data hit percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBdm% \fR
+.ad
+.RS 14n
+Demand data miss percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBmfu \fR
+.ad
+.RS 14n
+MFU List hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBmh% \fR
+.ad
+.RS 14n
+Metadata hit percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBmm% \fR
+.ad
+.RS 14n
+Metadata miss percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBmru \fR
+.ad
+.RS 14n
+MRU List hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBph% \fR
+.ad
+.RS 14n
+Prefetch hits percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBpm% \fR
+.ad
+.RS 14n
+Prefetch miss percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBdhit \fR
+.ad
+.RS 14n
+Demand data hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBdmis \fR
+.ad
+.RS 14n
+Demand data misses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBhit% \fR
+.ad
+.RS 14n
+ARC Hit percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBhits \fR
+.ad
+.RS 14n
+ARC reads per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBmfug \fR
+.ad
+.RS 14n
+MFU Ghost List hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBmhit \fR
+.ad
+.RS 14n
+Metadata hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBmiss \fR
+.ad
+.RS 14n
+ARC misses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBmmis \fR
+.ad
+.RS 14n
+Metadata misses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBmrug \fR
+.ad
+.RS 14n
+MRU Ghost List hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBphit \fR
+.ad
+.RS 14n
+Prefetch hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBpmis \fR
+.ad
+.RS 14n
+Prefetch misses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBread \fR
+.ad
+.RS 14n
+Total ARC accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBtime \fR
+.ad
+.RS 14n
+Time
+.RE
+
+.sp
+.ne 2
+.na
+\fBarcsz \fR
+.ad
+.RS 14n
+ARC Size
+.RE
+
+.sp
+.ne 2
+.na
+\fBdread \fR
+.ad
+.RS 14n
+Demand data accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBeskip \fR
+.ad
+.RS 14n
+evict_skip per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBmiss% \fR
+.ad
+.RS 14n
+ARC miss percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBmread \fR
+.ad
+.RS 14n
+Metadata accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBpread \fR
+.ad
+.RS 14n
+Prefetch accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2hit% \fR
+.ad
+.RS 14n
+L2ARC access hit percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2hits \fR
+.ad
+.RS 14n
+L2ARC hits per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2miss \fR
+.ad
+.RS 14n
+L2ARC misses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2read \fR
+.ad
+.RS 14n
+Total L2ARC accesses per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2size \fR
+.ad
+.RS 14n
+Size of the L2ARC
+.RE
+
+.sp
+.ne 2
+.na
+\fBmtxmis \fR
+.ad
+.RS 14n
+mutex_miss per second
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2bytes \fR
+.ad
+.RS 14n
+bytes read per second from the L2ARC
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2miss% \fR
+.ad
+.RS 14n
+L2ARC access miss percentage
+.RE
+
+.sp
+.ne 2
+.na
+\fBl2asize \fR
+.ad
+.RS 14n
+Actual (compressed) size of the L2ARC
+.RE
+
+.sp
+.ne 2
+.na
+\fBgrow \fR
+.ad
+.RS 14n
+ARC Grow disabled
```

Comment:
It looks like there are a handful of places inconsistently capitalized in the `arcstat -v` output and the man page descriptions.  Can you please update both locations, i.e `Hit -> hit`, `List -> list`, `Grow -> grow`.

Reply:
Done! I originally left them since I don't really know anything about ARC internals and wasn't sure if these were supposed to be capitalized